bot/exts/evergreen/bookmark.py

```diff
@@ -1,21 +1,89 @@
+import asyncio
 import logging
 import random
 
 import discord
 from discord.ext import commands
 
-from bot.constants import Colours, ERROR_REPLIES, Emojis, Icons
+from bot.bot import Bot
+from bot.constants import Colours, ERROR_REPLIES, Icons
 from bot.utils.converters import WrappedMessageConverter
 
 log = logging.getLogger(__name__)
 
+# Number of seconds to wait for other users to bookmark the same message
+TIMEOUT = 120
+BOOKMARK_EMOJI = "📌"
+
 
 class Bookmark(commands.Cog):
     """Creates personal bookmarks by relaying a message link to the user's DMs."""
 
-    def __init__(self, bot: commands.Bot):
+    def __init__(self, bot: Bot):
         self.bot = bot
 
+    @staticmethod
+    def build_bookmark_dm(target_message: discord.Message, title: str) -> discord.Embed:
+        """Builds the embed to DM the bookmark requester."""
```

Comment:
the Python Discord style guide tells to phrase the docstrings in an imperative way, i.e.
> Build the embed to DM the bookmark requester.